src/pycolmap/sfm/incremental_mapper.cc

```diff
@@ -360,6 +361,14 @@ void BindIncrementalMapperImpl(py::module& m) {
       .def_readwrite("num_adjusted_observations",
                      &LocalBAReport::num_adjusted_observations);
   MakeDataclass(PyLocalBAReport);
+  
+  py::class_<IncrementalMapperImpl>(m, "IncrementalMapperImpl")
```

Comment:
We shouldn't expose the impl class here. Instead, if we really need the find_local_bundle, let's expose it in the main incremental mapper and then bind the method there.